src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useTheme } from '../../context';
+import './navbar.scss';
+import { useAuth } from '../../context';
+import { makeToast } from '../../components';
+import NOTING_ICON from '../../assets/noting-icon.png';
+
+const Navbar = () => {
+  const navigate = useNavigate();
+  const { theme, toggleTheme } = useTheme();
+  const { authState, authDispatch } = useAuth();
+  const handleLogout = () => {
+    makeToast('You Are Now Logged Out', 'success');
```

Comment:
showing of this toast should be responsibility of auth context instead of some component.